add type

Dragonbuf opened the pull request:
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | yes
| Breaks BC?    | yes
| Tests pass?   | yes
| Fixed issues  | #363

Commit:
add type

samdark commented on src/cli/Command.php:
```diff
@@ -182,7 +182,7 @@ protected function handleMessage($id, $message, $ttr, $attempt)
         }
 
-        $process = new Process($cmd, null, null, $message, $ttr);
+        $process = new Process($cmd, (string)null, (array)null, $message, (float)$ttr);
```
How about

```suggestion
        $process = new Process($cmd, '', [], $message, (float)$ttr);
```

Commit:
in vendor/symfony/process/Process.php:157
 ,if $cwd default is '',will throw  new RuntimeException(sprintf('The provided cwd "%s" does not exist.', $this->cwd));

rob006 reviewed (commented):
This change does not make sense to me. #363 was about lack of support `symfony/process` v5.x - `TypeError: Argument 1 passed to Symfony\Component\Process\Process::__construct()` error was fixed a long ago, but OP installed an old version of `yii2-queue` because of invalid composer constraints in old releases. #363 should be fixed by adjusting `composer.json` to allow `symfony/process` v5. Cast to float should not be necessary (and definitely not related to #363)
